packages/core/types/index.js

```diff
@@ -670,19 +880,60 @@ export interface BundleGraph {
   ): ?TContext;
 }
 
+/**
+ * A BundleGraph in the Bundler that can be modified
+ * @method addAssetGraphToBundle Add asset and all child nodes to the bundle VERIFY how??
+ * @method createAssetReference FIXME
+ * @method createBundleGroup Turns an edge (Dependency -> Asset-s) into (Dependency -> BundleGroup -> Asset-s)
+ * @method getDependencyAssets FIXME a dependency can have multiple child nodes?
+ * @method removeAssetGraphFromBundle Remove all "contains" edges from the bundle to the nodes in the asset's subgraph.
+ * @method internalizeAsyncDependency Turns a dependency to a different bundle into a dependency to an asset inside <code>bundle</code>.
+ * @method traverse FIME difference to traverseContents?
+ * @method traverseContents FIXME
```

Comment:
1. the "references" edge again
2. `getDependencyAssets`?
3. traverse and traverseContents seem to do the same thing?